lab9.md

```diff
@@ -64,4 +64,60 @@ Now you can verify using `$ docker service ps nodeinfo` that new replicas of `no
 
 Now stop the bash script and you will see the replica count return to 1 replica after a few seconds.
 
+### Try scale from zero
+
+If you scale down your function to 0 replicas, you can still invoke it. The gateway will trigger your invocation and scale back the function to 1.
+
+Try it out with the following commands:
+
+Docker Swarm:
+```
+$ docker service scale nodeinfo=0
+```
+
+Kubernetes:
+```
+$ POD=$( kubectl get pods  -n openfaas-fn -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep astronaut-finder)
+$ kubectl scale --replicas=0 $POD -n openfaas-fn
+```
+
+Open the OpenFaaS UI and check that nodeinfo has 0 replicas, or by `docker service ls | grep nodeinfo`.
+
+> For Kubernetes use `kubectl get pods -n openfaas-fn | grep nodeinfo`
+
+Now invoke the function and check back that it scaled to 1 replicas.
+
+### Setup faas-idler
```

Comment:
I think this is OK, but we should make it an optional / advanced lab towards the end of the material. @rgee0 would you agree?

Reply:
I'm tempted to ask for this in a new Lab 12, and we should check but I think the idler is now part of the stack for both Swarm + K8s.